types/asana/index.d.ts

```diff
@@ -1741,10 +1893,16 @@ declare namespace asana {
              */
             searchInWorkspace(
                 workspace: number | string,
-                params?: Params,
+                params?: { opt_fields?: never } & { [key: string]: string },
+                dispatchOptions?: any,
+            ): Promise<ResourceList<Tasks.CompactType>>;
+            searchInWorkspace(
+                workspace: number | string,
+                params: { opt_fields: string } & { [key: string]: string },
```

Comment:
Params was wrong for this one; there's a variety of dynamically named properties that can be used (and must be provided) in a search: https://developers.asana.com/docs/search-tasks-in-a-workspace